gerrit-entrypoint.sh

```diff
@@ -168,7 +168,13 @@ if [ "$1" = "/gerrit-start.sh" ]; then
   [ -z "${HTTPD_LISTENURL}" ] || set_gerrit_config httpd.listenUrl "${HTTPD_LISTENURL}"
 
   #Section gitweb
-  set_gerrit_config gitweb.cgi "/usr/share/gitweb/gitweb.cgi"
+  if [ "${GITWEB_TYPE}" = 'gitiles' ]  ; then
```

Comment:
I'v never used gitiles with gerrit before. Have you ever tested this setting? Since I can't find this value in gerrit document's [gitweb.type](https://gerrit-review.googlesource.com/Documentation/config-gerrit.html#gitweb.type) part.